scheduler-jobs/README.md

```diff
@@ -0,0 +1,17 @@
+# How-To Run Locally:
+
+To run locally as a process: `go run scheduler-jobs.go`, assuming you have a scheduler and dapr running accordingly (see below).
+
+Run Scheduler:
+![scheduler_debugger_config.png](scheduler_debugger_config.png)
+
+Run Dapr sidecar:
+![sidecar_debugger_config.png](sidecar_debugger_config.png)
```

Comment:
Same as above, but update the app port to 3006.

Reply:
can you clarify which above comment you are referring to?

Reply:
I was referring to the one about showing the cli command to start the scheduler instead of a golang screenshot.
The comment about the port is that the image shows app port set to 8383, but the code runs the grpc server on 3006. Feel free to remove if you don't think it's needed, or update the screenshot, I'm fine with either.